packages/beacon-node/src/util/dataColumns.ts

```diff
@@ -139,8 +139,19 @@ export function getValidatorsCustodyRequirement(
     return total + state.epochCtx.effectiveBalanceIncrements[validatorIndex] * EFFECTIVE_BALANCE_INCREMENT;
   }, 0);
 
-  const count = Math.floor(totalNodeEffectiveBalance / config.BALANCE_PER_ADDITIONAL_CUSTODY_GROUP);
-  return Math.min(Math.max(count, config.VALIDATOR_CUSTODY_REQUIREMENT), NUMBER_OF_CUSTODY_GROUPS);
+  // Must custody one group for every BALANCE_PER_ADDITIONAL_CUSTODY_GROUP of effective balance
+  let validatorsCustodyRequirement = Math.floor(
+    totalNodeEffectiveBalance / config.BALANCE_PER_ADDITIONAL_CUSTODY_GROUP
+  );
+
+  // Any node with at least 1 validator must custody at least VALIDATOR_CUSTODY_REQUIREMENT
+  validatorsCustodyRequirement = Math.max(validatorsCustodyRequirement, config.VALIDATOR_CUSTODY_REQUIREMENT);
```

Comment:
not related to this PR but shouldn't the spec constant be called `MIN_VALIDATOR_CUSTODY_REQUIREMENT`?

Reply:
I agree that `MIN_` would make it clearer, but at least for now it's [`VALIDATOR_CUSTODY_REQUIREMENT`](https://github.com/ethereum/consensus-specs/blob/85e6451072531b6bc00a66e301f0636711f2886c/specs/fulu/validator.md?plain=1#L47)

Reply:
seems like spec has a clear description of the constant and we have good comments in the code as well, so might not be worth changing the constant name on the spec